presto-product-tests/pom.xml

```diff
@@ -83,8 +83,18 @@
     <build>
         <resources>
             <resource>
-                <directory>${project.basedir}/src/main/resources</directory>
+                <directory>src/main/resources</directory>
```

Comment:
Path is relative to `${project.basedir}` by default. I don't think that we need to specify it explicitly.